### DISCUSSION TITLE
救命，给我干哪来了，这还是六分街吗？

### DISCUSSION THREAD

**Discussion by Xyy12345689** [share121/inter-knot#4552] [其他]
![Screenshot_2024-09-07-20-23-36-22_f873a713f3f3d5709dc764418741c4f2](https://github.com/user-attachments/assets/6aa8931b-0fff-4e65-9936-3d021d595aac)
刚刚接到委托进入空洞，然后莫名其妙就晕了，醒来发现躺在一片草地上，身边还多了个紫色的，图案像电一样的玩意，等等，我好像看见远处有个黄毛在找东西，我去找他问问路

**Comment by NEBrightULA** [comment]
听说带着一些四角星形状的漂亮石头当作问路费会得到非常热情有效的帮助。
